layers/auto-completion/packages.el

```diff
@@ -269,7 +269,7 @@
         "Call `yas-expand' and switch to `insert state'"
         (interactive)
         (call-interactively 'aya-expand)
-        (evil-insert-state))
+        (unless holy-mode (evil-insert-state)))
```

Comment:
@StreakyCobra @nixmaniack There is no need to change anything for the current version of holy-mode either. In the current version of holy mode `(evil-insert-state)` puts you in hybrid state.